source/gridfs/tests/README.rst

```diff
@@ -1,66 +1,37 @@
+============
```

Comment:
Let me know if this file needs a changelog. I didn't think it was necessary as it was basically rewritten from scratch, but perhaps we'd like it in place for future use.

Reply:
I don't see any advantage with an empty changelog, so feel free to omit it.